goldstone/core/models.py

```diff
@@ -2634,3 +2634,38 @@ def integration(cls):
     def resourcetype(cls):
 
         return "routers"
+
+
+class EventQueryDef(PolyResource):
+
+    """ This model defines how events get generated from parsing
+        queries and log indices """
```

Comment:
Per PEP257, the docstring should be:

    """This model defines how events get generated from parsing
       queries and log indices."""